frontend/src/components/common/SimpleTable.tsx

```diff
@@ -42,6 +44,7 @@ interface SimpleTableColumn {
   cellProps?: {
     [propName: string]: any;
   };
+  sortFunction?: (isIncreasingOrder: boolean) => (arg1: any, arg2: any) => number;
```

Comment:
Another thing we could do to make this even more automatic is to provide a default sorting function and try to apply it to at least strings/numbers. E.g. we could use `sort` instead of `sortFunc` and it'd be either the `sortFunc` type you had, and `| boolean`.
If the `sort` prop is `true`, then we just use the default one (this should only be set for columns we are using "basic" values in, that we know our default sort function can consule), so either the `datum` or the result of the `getter` should be a known/sortable one. This requires a bit of work but maybe saves a lot of code/maintenance?